File: spec/code_samples/Php_http1/billings/post.php
<?php

$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/billings');
$request->setMethod(HTTP_METH_POST);

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE',
  'content-type' => 'application/json',
  'accept' => 'application/json'
));

$request->setBody('{"billing_address":{},"cc_data":{},"check_data":{},"promo_code":"<ADD STRING VALUE>"}');

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}

File: spec/code_samples/Python_requests/postings@custom_categories@{category_type}/post.py
import requests

url = "https://www.directfreight.com/api/postings/custom_categories/%7Bcategory_type%7D"

headers = {
    'accept': "application/json",
    'content-type': "application/json",
    'end-user-token': "SOME_STRING_VALUE"
    }

response = requests.request("POST", url, headers=headers)

print(response.text)